ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -60,17 +60,21 @@ import           Ouroboros.Consensus.Util.Orphans ()
 import           Ouroboros.Consensus.Util.Random
 import           Ouroboros.Consensus.Util.STM
 
--- TODO: We currently stil import /some/ stuff from the network layer Node
--- module. We should audit this and perhaps move these to different modules
--- in the networking layer to make the division clearer.
-
-import           Ouroboros.Network.Node (NodeId (..))
-import qualified Ouroboros.Network.Node as Network
 
 {-------------------------------------------------------------------------------
   Node IDs
 -------------------------------------------------------------------------------}
 
+-- TODO: This was moved here from the network layer Node module. We should
+-- review this and make sure it makes sense here.
+data NodeId = CoreId Int
+            | RelayId Int
+  deriving (Eq, Ord, Show)
```

Comment:
I suspect that we will eventually want to replace this with a different kind of node identifier, but for now this is fine.